### PR TITLE
Update WTS workflow inputs (dev and prod)

### PR DESCRIPTION
**Pull request by skanwal**
**Updated:**

- input for multiQC reference samples
- input for multiQC config

### CODE REVIEW

File: terraform/stacks/umccr_data_portal/workflow/main.tf [hunk 1]
```diff
@@ -307,30 +303,26 @@ locals {
...
           },
           {
               "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00199_MDX190193_L1900920/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00199"
+              "location": "gds://production/analysis_data/SBJ01147/wts_tumor_only/2022071381c4257a/L2101521_dragen"
           },
           {
               "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00236_MDX190219_L1901023/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00236"
+              "location": "gds://production//analysis_data/SBJ01620/wts_tumor_only/2022042945b6e532/L2200188_dragen"
```
Comment (victorskl) [inline terraform/stacks/umccr_data_portal/workflow/main.tf:314]: Typo: extra `/` in the path.

Also, side note -- do you recommend we clean up the "Mar 13 2022" early run transcriptome result for this https://data.umccr.org/subjects/SBJ01620?

**Review by victorskl — COMMENTED** [review]
One minor typo. Looking good otherwise. I can/will right away apply to DEV as soon as merge.

For PROD, can this go with next Kinkalow release?

File: terraform/stacks/umccr_data_portal/workflow/main.tf (hunk 1 above)
Comment (victorskl) [reply terraform/stacks/umccr_data_portal/workflow/main.tf:314]: Just realise that Run 84 has repeat due to 10X samples reconversion. Hence marked for cleanup on early run. So no worry about it.

**Comment by alexiswl** [comment]
Ah does this have any implications of we decide to archive output data at any point? Should this data instead be copied over to validation data in case directories the analysis data folder are one day archived after a certain date? @reisingerf

**Comment by reisingerf** [comment]
Hm, good point @alexiswl 

This is listed under reference data, so I guess should be in the validation data location. 

There's the slight issue that our reference data is not only public data, but also private patient data... not sure if we have any SOPs on how to handle/protect reference data, but that's another issue I'd say. In the worst case we'd need a public and a private reference data location...

As I see it, this data is used as a reference and used by workflows independent/unaware of the age of the data, both points speak against pointing to analysis data as location.

File: terraform/stacks/umccr_data_portal/workflow/main.tf (hunk 1 above)
Comment (skanwal) [reply terraform/stacks/umccr_data_portal/workflow/main.tf]: Fixed typo in https://github.com/umccr/infrastructure/commit/838a0d868b45663ac7926fadbc596a083754815f

**Comment by skanwal** [comment]
@alexiswl @reisingerf - I did consider moving/copying data to `validation_data`, but this set is not really a validation dataset for WTS workflow but reference data specific to multiQC. Hence, we might need to consider using `reference_data` folder for such datasets?

**Comment by skanwal** [comment]
Although, happy to make a change if "validation_data" (may be under `multiqc`) is considered as an appropriate location for this data.

**Comment by reisingerf** [comment]
Hm... if it's more reference than validation data, then reference_data may be the right location.
Same principals apply though...  perhaps one to open up to a wider audience?

**Comment by skanwal** [comment]
Sure, may be we can discuss in the team meeting or also happy to join devops call

**Comment by reisingerf** [comment]
Lab meeting sounds good

**Comment by victorskl** [comment]
- Lifted blocker tag
- Sehrish is working on re-arranging reference data location
- Related internal Trello card -- https://trello.com/c/JqCqnHbj/1238-update-wts-multiqc-reference-samples

**Comment by skanwal** [comment]
@victorskl @alexiswl - I have regenerated and updated the path to QC reference data on GDS.
If this looks fine, can one of you copy this data over to prod?

**Comment by alexiswl** [comment]
Okay if I migrate this from 
```
gds://development/reference-data/multiqc/
```

to

```
gds://development/reference-data/dragen_wts/multiqc/
```

Since all of these multiqc samples are for WTS? 

Can then add them to `gds://production/reference-data/dragen_wts/multiqc/`

**Comment by skanwal** [comment]
@alexiswl - no objections from my side. Thanks!

**Comment by alexiswl** [comment]
Migrating to 
`gds://development/reference-data/dragen_wts/multiqc/` and
`gds://production/reference-data/dragen_wts/multiqc/`

**Comment by alexiswl** [comment]
```
$ ica-context-switcher --scope admin --project-name production
Successfully switched to ICA project "production" with scope level "admin"

$ gds-ls gds://production/reference-data/dragen_wts/multiqc/
gds://production/reference-data/dragen_wts/multiqc/2022-08-12--3.9.3/
gds://production/reference-data/dragen_wts/multiqc/2022-08-17--3.9.3/
gds://production/reference-data/dragen_wts/multiqc/_manifest.json
gds://production/reference-data/dragen_wts/multiqc/_tags.json

$ ica-context-switcher --scope admin --project-name development
Successfully switched to ICA project "development" with scope level "admin"

$ gds-ls gds://development/reference-data/dragen_wts/multiqc/
gds://development/reference-data/dragen_wts/multiqc/2022-08-12--3.9.3/
gds://development/reference-data/dragen_wts/multiqc/2022-08-17--3.9.3/
gds://development/reference-data/dragen_wts/multiqc/_manifest.json
gds://development/reference-data/dragen_wts/multiqc/_tags.json
```

File: terraform/stacks/umccr_data_portal/workflow/main.tf [hunk 2]
```diff
@@ -243,32 +243,28 @@ locals {
...
-          },
-          {
-              "class": "Directory",
-              "location": "gds://development/validation_data/wts/SBJ00028_PRJ190190_L1900423/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00028"
-          }
-      ],
-      "cl_config": "{\"sample_names_replace\":{\"L1900440\":\"Ref_1\",\"L1900877\":\"Ref_2\",\"L1900920\":\"Ref_3\",\"L1901023\":\"Ref_4\",\"L1901028\":\"Ref_5\",\"L1900423\":\"Ref_6\"}}",
+            {
+                "class": "Directory",
+                "location": "gds://development/reference-data/multiqc/2022-08-17--3.9.3/SBJ01563"
```
Comment (victorskl) [inline terraform/stacks/umccr_data_portal/workflow/main.tf:248]: Ah, need mid-path `/dragen_wts/` update here,  now.

File: terraform/stacks/umccr_data_portal/workflow/main.tf [hunk 3]
```diff
@@ -305,32 +301,28 @@ locals {
...
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00028_PRJ190190_L1900423/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00028"
-          }
-      ],
-      "cl_config": "{\"sample_names_replace\":{\"L1900440\":\"Ref_1\",\"L1900877\":\"Ref_2\",\"L1900920\":\"Ref_3\",\"L1901023\":\"Ref_4\",\"L1901028\":\"Ref_5\",\"L1900423\":\"Ref_6\"}}",
+            {
+                "class": "Directory",
+                "location": "gds://production/reference-data/multiqc/2022-08-17--3.9.3/SBJ01563"
```
Comment (victorskl) [inline terraform/stacks/umccr_data_portal/workflow/main.tf:306]: Yep, ditto mid-path `/dragen_wts/`

**Review by victorskl — COMMENTED** [review]
Sehrish, would it be quick to update mid-path from your side, pls?

File: terraform/stacks/umccr_data_portal/workflow/main.tf (hunk 3 above)
Comment (skanwal) [reply terraform/stacks/umccr_data_portal/workflow/main.tf]: updated in [bc39467](https://github.com/umccr/infrastructure/pull/243/commits/bc39467cb32d623fc7548cbb8354c283007edd07)

File: terraform/stacks/umccr_data_portal/workflow/main.tf (hunk 2 above)
Comment (skanwal) [reply terraform/stacks/umccr_data_portal/workflow/main.tf]: updated in [bc39467](https://github.com/umccr/infrastructure/pull/243/commits/bc39467cb32d623fc7548cbb8354c283007edd07)

**Review by victorskl — APPROVED** [review]
Awesome teamwork, folks! Happy to merge this. :)

**Review by victorskl — APPROVED** [review]
FYI, merging... :)

**Pull request merged**